strings/java_strings_intruduction/Solution.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+import java.lang.String;
+
+public class Solution {
+  
+  private static int lengthSum(String firstStringToSum, String secondStringToSum) {
+    return (firstStringToSum.length() + secondStringToSum.length());
+  }
+  
+  private static String compareStringLexicographically(String firstStringToCompare, String secondStringToCompare) {
```

Comment:
return bool and simplify signature. Suggestion:
`String isLess (String first, String second)`